src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final String DEFAULT_COLOR = Color.WHITE.name().toLowerCase();
+    private static final int DEFAULT_RADIUS = 10;
+    private static final int FIGURE_COUNT = 5;
+    private static final int MAX_LENGTH = 50;
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Figure figure;
+
+    private int randomSize() {
+        return random.nextInt(MAX_LENGTH);
+    }
+
+    public Figure getDefaultFigure() {
+        return new Circle(DEFAULT_RADIUS, DEFAULT_COLOR);
```

Comment:
The DEFAULT_COLOR should be obtained directly from the enum without calling name() and toLowerCase() (line 6). Enums are constants, and their string representation should be in uppercase to follow Java naming conventions.